geom/src/lib.rs

```diff
@@ -53,3 +53,38 @@ pub struct UnitFmt {
     /// Display in metric; US imperial otherwise.
     pub metric: bool,
 }
+
+#[derive(Clone, Copy, Debug)]
+pub struct CornerRadii {
```

Comment:
This is to solve things like this:

<img width="146" alt="Screen Shot 2021-01-24 at 5 08 21 PM" src="https://user-images.githubusercontent.com/217057/105646837-b5393600-5e67-11eb-9c7c-d27747cb3511.png">

(note the gap near the corners of  the triangle button)

We sometimes want unequal rounding across corners. This lets you specify that. It's kind of a heavy-handed solution, but I'm not sure of a better way.

Reply:
Note I'm levaraging `Into` implementations so that most things continue to work as before.